tutorials/roofit/rf614_bias_in_binned_fits.C

```diff
@@ -0,0 +1,195 @@
+/// \file
+/// \ingroup tutorial_roofit
+/// \notebook -js
+/// A tutorial that explains you how to solve problems with binning effects and
+/// numerical stability in binned fits.
+///
+/// ### Introduction
+///
+/// In this tutorial, you will learn three new things:
+///
+///  1. How to reduce the bias in binned fits by changing the definition of the
+///     normalization integral
+/// 
+///  2. How to completely get rid of binning effects by integrating the pdf over each bin
+///
+///  3. How to improve the numeric stability of fits with a greatly different
+///     number of events per bin, using a constant per-bin counterterm
+///
+/// \macro_code
+/// \macro_output
+///
+/// \date January 2023
+/// \author Jonas Rembser
+
+// Generate binned Asimuv dataset for a continuouts pdf.
```

Comment:
Asimuv -> Asimov

Reply:
Here also you are creating a data set with expected events. In generated binned the values for each bin will be generated according to a Poisson process. In principle you could add a gRandom->Poisson(nEvents * integ->getVal()) to have the correct result of `generatedBinned`

Reply:
It is the intention that the dataset has the expected number of events. The `generateBinned()` function would do the same if you don't specify the `Extended()` command argument.